tests/README.md

```diff
@@ -0,0 +1,25 @@
+# Marathon Testing
+
+The Marathon project has several styles of tests that are maintained with the project that include:
+
+* Unit Testing + Mocking
+* System Testing
+
+##  Unit Testing
+
+The unit and mocking tests are part of the project.  The test code is managed in the
+code base under `src/test` and are involved with `sbt test`.  These tests are whitebox tests and
+are focused on testing the internals of Marathon.
+
```

Comment:
We also have `sbt integration:test` and benchmarking with `sbt bench:test`. The integration tests generally spawn additional processes and take significantly longer.